plugins/UsersManager/API.php

```diff
@@ -1720,7 +1720,7 @@ public function generateInviteLink(
             ]);
     }
 
-    private function executeConcurrencySafe(string $userLogin, callable $callback = null)
+    private function executeConcurrencySafe(string $userLogin, ?callable $callback = null): void
```

Comment:
```suggestion
    private function executeConcurrencySafe(string $userLogin, callable $callback): void
```

There is no usage without a callback. And the `Lock` itself does not declare the callback as optional.

Reply:
Hm. You're right. Makes sense to change it then.